### ISSUE TITLE
🤖 [Dynamic Cards] Open reader deep links without switching tabs

### ISSUE THREAD

**Issue by antonis** [wordpress-mobile/WordPress-Android#19825]
It be nice to not redirect users to the reader tab and just open a post when a post deep link is used in a dynamic card (reference https://github.com/wordpress-mobile/WordPress-Android/pull/19810#pullrequestreview-1789493936)

**Comment by antonis** [comment]
I've spent some time investigating this today but wasn't able to find a way to circumvent the current default deep linking mechanism that by default switches to the reader tab. Note that this is the expected behavior on all other cases.
A workaround might involve adding a "new type of reader deep link" in the dynamic card url so that it is handled in a different way. Alternatively we could construct this "new type of reader deep link" as a special case when we detect a reader post link in the dynamic dashboard card url on the client side.

**Comment by justtwago** [comment]
> Alternatively we could construct this "new type of reader deep link" as a special case when we detect a reader post link in the dynamic dashboard card url on the client side.

Yeah, I'd give it a try when we have some free time :) Thank you for the investigation!
